scripts/onboarding/aks/onboarding-using-msi-auth/existingClusterOnboarding.json

```diff
@@ -85,6 +85,18 @@
             "metadata": {
```

Comment:
do we need to support bicep/terraform and azure policy of this?

Reply:
> do we need to support bicep/terraform and azure policy of this?

>Yes, I will make separate PR for other onboarding methods.